packages/mermaid/src/diagrams/gantt/ganttDb.js

```diff
@@ -298,7 +359,7 @@ const getStartDate = function (prevTime, dateFormat, str) {
       d.getFullYear() < -10000 ||
       d.getFullYear() > 10000
     ) {
-      throw new Error('Invalid date:' + str);
+      throw new Error('Invalid date: `' + str + '` with date format: `' + dateFormat + '`');
```

Comment:
Use string templates. 
```suggestion
      throw new Error(`Invalid date: '${str}' with date format: '${dateFormat}'`);
```